### DISCUSSION TITLE
Flower configuration

### DISCUSSION THREAD

**Discussion by dehume-drizly** [dagster-io/dagster#2960] [Q&A] [answered]
Is there a way to config and launch flower using the same parameters as your `celery.yaml` that you can with your worker?

**Comment by johannkm** [comment, marked as answer]
Hi @dehume-drizly, it looks like currently we only place the broker and backend. Are you hoping to pass the `configSource` from your `Values.yaml`?

**Reply by dehume-drizly** [reply]
I was using a custom Dagster celery app with (SQS and Dynamo) and thought I needed `configSource` but I think I can actually get by with just the broker and backend. So I should be good.
